pkg/551cb44b-5b4c-43c4-84ee-9044327b50c8/package.lp

```diff
@@ -0,0 +1,136 @@
+(librepcb_package 551cb44b-5b4c-43c4-84ee-9044327b50c8
+ (name "TCA9548A Breakout")
+ (description "Low-Voltage 8-Channel I2C Switch with Reset breakout board.")
+ (keywords "i2c,mux,multiplexer,adafruit,breakout")
+ (author "Marc Wittwer")
+ (version "0.1")
+ (created 2019-06-11T12:04:24Z)
+ (deprecated false)
+ (category 56a5773f-eeb4-4b39-8cb9-274f3da26f4f)
```

Comment:
The category "Connector" doesn't match for a breakout board. Since the breakout board is very specific, I would rather add a new package category "Manufacturer-Specific -> Adafruit" and add the breakout board there. See for example the Luminus library which does the same.